### ISSUE TITLE
A pure WebAssembly / JavaScript port of FFmpeg

### ISSUE THREAD

**Issue by hashg** [mifi/editly#90]
- [ ] I have tried with the newest version of editly: `npm i -g editly`
- [ ] I have tried ffmpeg newest stable version
- [ ] I have searched for existing issues

Please checkout https://ffmpegwasm.github.io/

**Comment by mifi** [comment]
Yes! But it looks like it doesn't support streaming. All files read/written need to be loaded into memory in full. This means it would not be very easy to integrate in editly.

**Comment by thebrucecgit** [comment]
I have no idea how WASM works so I could be wrong, the front page of that website says:

> It enables video & audio record, convert and stream right inside browsers.

Is that different to what you mean by streaming?

**Comment by mifi** [comment]
I mean itlooks like it doesn't support streaming data from `stdin` (which is how editly works). It needs to stream from a stored file (in memory)

**Comment by haakonstorm** [comment]
I might very well be way off here, but could combining shell process substitution with input redirection perhaps work? `< <(list)`
